clang/lib/StaticAnalyzer/Core/LoopUnrolling.cpp

```diff
@@ -226,6 +226,17 @@ static bool isPossiblyEscaped(ExplodedNode *N, const DeclRefExpr *DR) {
           return false;
       }
     }
+
+    if (const SwitchStmt *SS = dyn_cast<SwitchStmt>(S)) {
+      for (const Stmt *CB : dyn_cast<CompoundStmt>(SS->getBody())->body()) {
+        for (const Decl *D : dyn_cast<DeclStmt>(CB)->decls()) {
```

Comment:
And this will crash on all statements that are within a switch and not `DeclStmt`s (because the `dyn_cast` returns a nullpointer and you're calling a method on it).

Reply:
Oh, that's a stupid mistake. Thanks again.